docs/dev/how-to/send-message-l2-l1.md

```diff
@@ -261,20 +162,20 @@ try {
 }
 ```
 
-@tab Python
+6. Add the following lines to your `package.json` in the root folder:
 
-```sh
-// In progress. Check back later.
+```json
+"scripts": {
+   "run-file": "ts-node file.ts"
+}
 ```
 
-@tab Go
+7. To run the script, execute:
 
-```sh
-// In progress. Check back later.
+```bash
+npm run run-file
```

Comment:
The step-by-step has been using yarn commands until now, better be consistent:

```suggestion
yarn run-file
```

Reply:
Fixed in the latest commit